discovery-infra/start_discovery.py

```diff
@@ -291,6 +303,84 @@ def validate_dns(client, cluster_id):
         raise e
 
 
+def configure_terraform_workers_nodes(tf_folder, num_worker_nodes):
```

Comment:
it is relevant only for 2_day right? maybe better to create python file dedicated for it?

Reply:
and what with second interface? @RazRegev ?